src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalPayment = 0;
+
+  const dailyPrice = 40;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+
+  if (days < 3) {
+    totalPayment = dailyPrice * days;
+  }
+
+  if (days >= 7) {
+    totalPayment = (dailyPrice * days) - maxDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
you don't need `days < 7` check here, if `days >= 7` will be false then `days < 7` always will be true